site/app/templates/autograding/PeerResults.twig

```diff
@@ -18,6 +20,27 @@ function open_peer_feedback_tab(user, ordered_graders) {
     }
     $('#peer_feedback_nav_'+user).addClass('active-btn');
 }
+function set_peer_feedback(grader_id, user_id, g_id, feedback) {
+    $('#peer_feedback_no_response_'+grader_id).css("background-color", "#ffffff");
+    $('#peer_feedback_thanks_'+grader_id).css("background-color", "#ffffff");
+    $('#peer_feedback_helpful_'+grader_id).css("background-color", "#ffffff");
+    $('#peer_feedback_detailed_'+grader_id).css("background-color", "#ffffff");
+    $('#peer_feedback_inappropriate_'+grader_id).css("background-color", "#ffffff");
+    $('#peer_feedback_no_response_'+grader_id).css("color", "#008000");
+    $('#peer_feedback_thanks_'+grader_id).css("color", "#008000");
+    $('#peer_feedback_helpful_'+grader_id).css("color", "#008000");
+    $('#peer_feedback_detailed_'+grader_id).css("color", "#008000");
+    $('#peer_feedback_inappropriate_'+grader_id).css("color", "#800000");
+    $('#peer_feedback_'+feedback+'_'+grader_id).css("color", "#ffffff");
```

Comment:
Could this just be include in the css for this page?

Reply:
Added to server.css